### ISSUE TITLE
PD: feedback

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/pv-solar-farm-forecasting#11]
[ukpn_dashboard_report.pdf](https://github.com/openclimatefix/pv-solar-farm-forecasting/files/10482142/ukpn_dashboard_report.pdf)

Thanks @vrym2 for sending me this. its looks really good

Feel free to take or leave this comments

- Quick spelling check might help, ive got something called `Grammarly` installed that seems to help me a bit. 
- Bit more text would be handy. A paragraph of text saying what the plot is, and what it shows. 
- It would be nice to know how often the data is normally. every 5 mins? every 5 seconds? 
- How many missing values are typical? E.g. Across 1 month there were 0.1% of missing values. 
- It would be great to see for the different GSP, the max, min, mean std e.t.c., perhaps a histogram plot. (you might want to remove the zeros, as they cause a funny spike)
- It would be great to know how live the data is? Is it only showing the last day? Perhaps the last hour of data can be got. 
- Personally, I prefer `plotly` for plotting, but you can try that out another time
- nice plot of sun rise and sun set, showing the funny night time data

**Comment by vrym2** [comment]
@peterdudfield 
Thank you for your suggestions and comments, I have prepared the data report for the above tasks, and I had a meet with Dan to explain and take any comments and suggestion regarding my work. He had an excellent observation regarding missing, and duplicated time intervals in the GSP. It was due to the daylight savings. I'll be including this observation with some plots to explain this phenomenon. I have added `Grammarly` to my VScode, but yet to figure out how to utilise that. Please leave more comments and suggestions at your convenience. I'll carry on with improving the report further!

[ukpn_dashboard_report.pdf](https://github.com/openclimatefix/pv-solar-farm-forecasting/files/10491919/ukpn_dashboard_report.pdf)

**Comment by peterdudfield** [comment]
Looks good, I like Dan's idea of doing it by Hour of day. Perhaps keep the seasonal plot in there as well. 

Could you give an example of when the GSP data is available from? Is it 2021, or 2010, good to know. 

I think you just use `Grammarly` to do some spelling checks. 

Would be great to add a few more day plots, maybe a typical summer day, typical winter day. 

Burwell has a surprising number of negative values, could you plot a typical day to show this? Just checking negative means `<0`, not `<=0`. you might want to consider `<0.0001` which gets rid of the noise around 0

**Comment by vrym2** [comment]
@peterdudfield 
Here is a refined version of the data report with more context, and neatly `syntaxed`. I have shown the `dataframe.head(5)` of a typical loaded csv file where you can see from when the data is availabe, also added typical summer and winter day plots. I'll carry on further finishing the remaining tasks from the PR. Please check and comment at your own convenience.
[ukpn_dashboard_report.pdf](https://github.com/openclimatefix/pv-solar-farm-forecasting/files/10511324/ukpn_dashboard_report.pdf)

**Comment by peterdudfield** [comment]
Looks good @vrym2 

Could you do a plot of one of the GSPs that has lots of negative values. 30% seems very high

Is it clear for you what to do next?

**Comment by vrym2** [comment]
Yes, I beleive, the tasks remaining from the other PR #8 is that 
1. Compare `PVLive` `gsp` data to the current `ukpn` `gsp`, 
2. adjust the timeseries, considering daylight savings
3. create a automatic download script for the automation(Bonus)

Sure, I'll plot that as well, `Rayleigh` gsp has almost 50-60% of negative values in the total dataset
